doc/userguide/install.rst

```diff
@@ -193,6 +199,8 @@ After installing you can proceed to the :ref:`Basic setup`.
 Upgrading
 """""""""
 
+.. note:: The following instructions require `sudo` be installed.
+
```

Comment:
As removing, upgrading or running Suricata would only be possible if one had previously installed it, I don't think we need the note about installing `sudo` or in the sections that are not about installing Suri.

Reply:
So you're suggesting the notes there are not necessary then.

You're right, still. 

OK.
Working on it now.

Reply:
😺